docs/src/pages/components/tabs/tabs.md

```diff
@@ -91,6 +91,29 @@ Tab labels may be either all icons or all text.
 
 {{"demo": "pages/components/tabs/IconLabelTabs.js", "bg": true}}
 
+## Accessibility
+
+(WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#tabpanel)
+
+This section lists necessary steps to provide necessary information for assistive technology:
+
+1. label `Tabs` via `aria-label` or `aria-labelledby`
+2. `Tab`s need to be connected to their
+corresponding `[role="tabpanel"]` by setting the correct `id`, `aria-controls` and `aria-labelledby`
+
+An example for the current implementation can be found in the demos on this page. We've also published an experimental API in `@material-ui/lab` that does not require
+extra work.
+
+### Keyboard navigation
+
+The components implement keyboard navigation using the "manual activation" behavior. If you want to switch to the
+"selection automatically follows focus" behavior you have pass `selectionFollowsFocus` to the `Tabs` component. The WAI-ARIA authoring practices have a detailed guide on [how to decide then to make selection automatically follow focus](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus).
```

Comment:
```suggestion
"selection automatically follows focus" behavior you have pass `selectionFollowsFocus` to the `Tabs` component. The WAI-ARIA authoring practices have a detailed guide on [how to decide when to make selection automatically follow focus](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus).
```

To which the answer seems to be "always", since there should be no reason why an asynchronous network request for tabpanel content should block tab navigation.

I recall a comment that the default for v5 is up for discussion, and that current behaviour is retained to avoid a breaking change,  but I think the wording in this section should more strongly advocate the use of this prop.

Reply:
Right now I would agree with this. However, it's likely that this could be harmful for certain UIs. I think promise cancellation isn't a thing right now. Automatic activation could kick of a bunch of network requests that each have the same priority. You'd switch 5 tabs and now the first 4 tabs are starving the fifth request. 

There's also the issue of render-blocking. Keyboard navigation is user-input and therefore any slight lag is noticed immediately. Since concurrent react is not stable yet, switching could introduce a sluggish UI if the corresponding panel takes more than 16ms to render. 

It's a tough decission. I'd say in concurrent react (and if the Offscreen API lands) `selectionFollowsFocus` is better for UX. Everyone else should test this in production first.

We can extend the section a bit more with things to consider (too many requests, render blocking etc).

Reply:
Good points well made, thanks!